packages/nimble-components/src/theme-provider/tests/tokens.stories.ts

```diff
@@ -43,6 +49,8 @@ const metadata: Meta = {
 
 export default metadata;
 
+DesignToken.registerRoot(document.documentElement);
```

Comment:
We shouldn't need this, the [elements being defined](https://github.com/microsoft/fast/blob/1be91f1b6212a55963cdaaa0cd8af76b4d7091fa/packages/web-components/fast-foundation/src/design-system/design-system.ts#L306) should trigger this for us, i.e. according to the [docs](https://www.fast.design/docs/design-systems/design-tokens/#usage-with-designsystem):

> If you're using DesignSystem to register components and dependencies, then note that DesignToken root registration happens automatically when DesignSystem.register() is invoked.

And the storybook bundle pulls in all the components, so I expect that the root is getting registered (even though this story in-specific doesn't pull them in, the bundled build will happen to).

But it does make me wonder why the [site page works which just imports the DesignTokens](https://github.com/ni/nimble/blob/df00b90b8b7f93a0c5e3e53d158a66da04877050/packages/site/landing/index.ts#L1) and no components.

It seems like in [design-tokens.ts](https://github.com/ni/nimble/blob/df00b90b8b7f93a0c5e3e53d158a66da04877050/packages/nimble-components/src/theme-provider/design-tokens.ts) at the start (or end?) we should call `DesignToken.registerRoot();` so that the tokens can be used standalone (even though they already seem usable that way for some reason).

Also calling it with `document.documentElement` I think changes the default target.. At least in source the [default target is `document`](https://github.com/microsoft/fast/blob/1be91f1b6212a55963cdaaa0cd8af76b4d7091fa/packages/web-components/fast-foundation/src/design-token/custom-property-manager.ts#L229) instead of `document.documentElement`.

I think we should call it without the parameter and use the default fallback (which was already in use) unless we have a reason to switch.

Reply:
Sorry for this confusion: including `registerRoot()` was an experiment I had tried which didn't pan out and I forgot to remove before posting the PR. It indeed was having no effect because FAST was calling it already internally. I've removed it in the latest update.

This is a good reminder to me to re-scan the diff before sending code owners on deep dives into the docs. Sorry!